extensions/ql-vscode/src/databases/local-databases/database-manager.ts

```diff
@@ -662,18 +663,19 @@ export class DatabaseManager extends DisposableObject {
     }
     // note that we use undefined as the item in order to reset the entire tree
     this._onDidChangeDatabaseItem.fire({
-      item: undefined,
+      item,
```

Comment:
There is actually nowhere that listens to the `add` event and requires this value to be defined. Just for consistency, I thought I'd change it.